deb-arm/Dockerfile

```diff
@@ -112,6 +120,21 @@ RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then \
 ENV PATH="/usr/local/go/bin:${PATH}"
 ENV PATH="${GOPATH}/bin:${PATH}"
 
+# Rust is needed to compile the SDS library
+RUN if [ "$DD_TARGET_ARCH" = "aarch64" ] ; then curl -sSL -o rustup-init https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${RUSTUP_ARCH_AARCH64}/rustup-init \
+    && echo "${RUSTUP_SHA256_AARCH64}  rustup-init" | sha256sum --check \
+    && chmod +x ./rustup-init \
+    && ./rustup-init -y --profile minimal --default-toolchain ${RUST_VERSION} \
+    && echo "${RUSTC_SHA256_AARCH64}  $HOME/.cargo/bin/rustc" | sha256sum --check \
+    && rm ./rustup-init; fi
+RUN if [ "$DD_TARGET_ARCH" = "armhf" ] ; then curl -sSL -o rustup-init https://static.rust-lang.org/rustup/archive/${RUSTUP_VERSION}/${RUSTUP_ARCH_ARMHF}/rustup-init \
+    && echo "${RUSTUP_SHA256_ARMHF}  rustup-init" | sha256sum --check \
+    && chmod +x ./rustup-init \
+    && ./rustup-init -y --profile minimal --default-toolchain ${RUST_VERSION} \
+    && echo "${RUSTC_SHA256_ARMHF}  $HOME/.cargo/bin/rustc" | sha256sum --check \
+    && rm ./rustup-init; fi
```

Comment:
I think this would be more concise if you were conditionally assigning the triplet/shasum to a common couple of variables and only had a single install instruction

Reply:
I'll do this.